### ISSUE TITLE
Datepicker: Not applying minutes with timeFormat and default intervals when minutes is not divisible by interval

### ISSUE THREAD

**Issue by fitzorama** [infor-design/enterprise#4230]
**Describe the bug**
When a datepicker is configured to use the timeFormat with the default minute and/or second intervals and time is current, the minutes and/or seconds will only be applied if the current minute and/or seconds are divisible by the interval.

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/datepicker/example-timeformat.html
2. Using the third field, Date Time Picker in Locale (Time to Current), open the datepicker 
3. The time portion of the picker should display the current time
4. Click Apply
4. The error occurs when the current minute and/or current second is not divisible by the interval, which for the default is 5, and when this happens the minute and/or second will be zero.

**Expected behavior**
Regardless of the interval, the current time should be applied.

**Version**

- ids-enterprise: v4.31.0-beta.1

**Screenshots**
![datepicker-time-interval](https://user-images.githubusercontent.com/20212079/89442195-0156b980-d714-11ea-8226-48bb53968d56.gif)

**Platform**
 - All

**Additional context**

**Comment by tmcconechy** [comment]
Not 100% sure about this one. But it could be fixed by using the interval of 1. I made a conscious choice on this case.
Maybe the current time should round to match the interval when it opens? Or do you think this case should totally ignore the interval setting?

cc @deep7102

**Comment by fitzorama** [comment]
That's a good question. I think if the time is using current time, it should ignore the interval. Although using the minute/second dropdown is the only way to 'change' the current time and that is limited by the interval.

I agree using an interval of 1 will work and we may change to use that. 

I think having the time round to match the interval would be confusing because it wouldn't match the device time, since it would either be behind or ahead.

**Comment by tmcconechy** [comment]
Yeah ok. I think i agree then. We'll just make it insert the time as is for this use case.

**Comment by jbrcna** [comment]
**QA Passed**
v4.32.0-dev
will retest once beta is deployed
![image](https://user-images.githubusercontent.com/56722184/91171510-28018200-e70d-11ea-8598-60ed8488b002.png)

![image](https://user-images.githubusercontent.com/56722184/91171453-13bd8500-e70d-11ea-8575-8f4b34f03c08.png)

**Comment by jbrcna** [comment]
Retest Done. Test Passed
v4.32.0-beta0
https://4320-beta0-enterprise.demo.design.infor.com/components/datepicker/example-timeformat.html
